src/dropdown/dropdown.js

```diff
@@ -23,6 +24,7 @@ angular.module('ui.bootstrap.dropdown', ['ui.bootstrap.position'])
   this.close = function(dropdownScope) {
     if (openScope === dropdownScope) {
       openScope = null;
+      $document.unbind('touch', closeDropdown);
```

Comment:
I close it because I realize that the thing that was wrong and nonfunctional were the $document.bind (mobilephone) in angular ~1.4.7 so my code does not fix anything... I am looking what is going on.